docker/Dockerfile

```diff
@@ -7,19 +7,21 @@ COPY ./go-tf-prepare/main.go main.go
 COPY ./go-tf-prepare/pkg/ pkg/
 RUN GOOS=linux GOARCH=amd64 GO111MODULE=on go build -o tf-prepare main.go
 
-FROM alpine:3.12.1
+FROM debian:bullseye-slim
+# 11.1-slim
```

Comment:
I think 11.1-slim is bullseye-slim but using a specific version. Please do that :)

Reply:
building the file didnt work with "11.1-slim".